dts/arm/nordic/nrf_common.dtsi

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/i2c/i2c.h>
+#include <dt-bindings/gpio/gpio.h>
+
+#include <arm/nordic/override.dtsi>
+/*
+ * This value can be overridden at the board level or in an application specific
+ * dts/arm/nordic/override.dtsi file.
+ */
+#ifndef DT_DEFAULT_IRQ_PRIORITY
+#define DT_DEFAULT_IRQ_PRIORITY 1
```

Comment:
This should be something like DT_NRF_DEFAULT_IRQ_PRIORITY, since this is not a generic feature.